src/index.html

```diff
@@ -12,6 +12,10 @@
     <script>
         window.web3 = new Web3(ULIFrameProvider.create('mainnet'))
     </script>
+    </br></br><input id='address' type='text' maxlength='42' style='width: 26em' placeholder='Receiver address'></input></br>
+    <button style='width: 26em;' onclick='web3.eth.requestAccounts().then(accounts => {console.log({accounts});web3.eth.sendTransaction({from: accounts[0], to: document.getElementById("address").value, value: 10000000000000000}).then(({transactionHash}) => console.log({transactionHash}))})'>
+        Send 0.01 ETH to inputed address
```

Comment:
`... to passed address`

Reply:
We don't need to `console.log` accounts. `console.log({transactionHash})` is also redundant.